teachopencadd/talktorials/T035_rnns/talktorial.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<p>Put a link to the Talktorial introducing one-hot encoding.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
One thing, I forgot: There's a review section on the top of the notebook, when finalizing, I'd remove/resolve that one as well.
But overall, a very nice and good notebook!